packages/components/tests/dummy/app/router.js

```diff
@@ -80,4 +80,7 @@ Router.map(function () {
   this.route('overrides', function () {
     this.route('power-select');
   });
+  this.route('demo-app', function () {
```

Comment:
I think there is a real value in having the `demo-app` around. I tried the second suggestion (not rendering the `shw-` elements in the `demo-app`) in 0298bd4f56cb4d339f0d38342917a595f010bbfe. It still shares the same `index.html` that contains the `<body class="showcase-app">` so one override is still needed, but we were able to remove the other ones. Happy to give the first suggestion a go as well, but more likely in a separate PR.

Reply:
> I think there is a real value in having the demo-app around.

Ok, we agree :)

> I tried the second suggestion (not rendering the shw- elements in the demo-app) in https://github.com/hashicorp/design-system/commit/0298bd4f56cb4d339f0d38342917a595f010bbfe. It still shares the same index.html that contains the <body class="showcase-app"> so one override is still needed, but we were able to remove the other ones. Happy to give the first suggestion a go as well, but more likely in a separate PR.

I would prefer to see this (including your workaround in https://github.com/hashicorp/design-system/commit/0298bd4f56cb4d339f0d38342917a595f010bbfe) done is a separate PR for a few reasons:
- we can discuss during our engineering team sync the options and come up with a shared solution
- I suspect this demo app will play a part in the future, becoming a second "showcase", for dynamic interactions with the components, and more real-use-cases example, so having it done "properly" (whatever that means) will pay us back in the long term
- leaves this PR a bit cleaner, and we decouple it from finding a solution to the `demo-app` problem (that could take longer) so that the PR can be merged straightaway

What do you think?

Reply:
Makes sense! Will cherry-pick them in a separate PR and remove them from here